#138: Removed Exasol Artifactory references from user guide (#139)
exasol/spark-connector: doc/user_guide/user_guide.md

to your `build.sbt` file:

```scala
libraryDependencies += "com.exasol" % "spark-connector" %% "<VERSION>"
```

> Similar please do not forget to update the `<VERSION>` placeholder.

Reviewer: ```suggestion
Similarly, please do not forget to update the `<VERSION>` placeholder.
```

Go to your cluster, then to `Libraries`, and click `Install New`:

- Select Maven as a `Library Source`

> - In the `Coordinates` field, enter artifact coordinates
>   `com.exasol:spark-connector_2.13:<VERSION>`. Please notice that we use the
>   Scala version 2.13, please let us know if you require Scala version 2.12.
> - Click `Install`

Reviewer: ```suggestion
- In the `Coordinates` field, enter artifact coordinates
  `com.exasol:spark-connector_2.13:<VERSION>`. Please note that we use the
  Scala version 2.13. Please contact us know if you require Scala version 2.12.
- Click `Install`
```

```sh
spark-submit \
  --master spark://spark-master-url:7077 \
  --packages com.exasol:spark-connector_2.13:<VERSION> \
  --class com.organization.SparkApplication \
  path/to/spark_application.jar
```

The `--packages` parameter can be omitted if your Spark application

> JAR already includes the connector as a dependency (e.g, jar-with-dependencies).

Reviewer: ```suggestion
JAR already includes the connector as a dependency (e.g, `jar-with-dependencies`).
```

The assembled jar file with the `-assembly` suffix should be located in the `target/`
folder.

If you want different version of Spark, you can use profiles `-Pspark3.2` or

> `-Pspark3.1` for Spark `3.2` or `3.1` versions respectively.

Reviewer: ```suggestion
`-Pspark3.1` for Spark `3.2` or `3.1` versions, respectively.
```
See https://linguaholic.com/linguablog/comma-before-respectively/

Author: Nice, didn't know it :))
